eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -280,7 +282,6 @@
 {% block ezgmaplocation_field %}
 {##
  # This field type block accepts the following parameters:
- # - string mapType the Google map type (ROADMAP, SATELLITE, HYBRID or TERRAIN), default is ROADMAP
  # - boolean showMap whether to show the Google map or not, default is true
```

Comment:
Drop "Google" from the sentence please :)
Also there's the reference to Google's JS API at the bottom of this comment block (can't point it here). Some updated link to OpenStreet doc would be good.